lib/menu.rb

```diff
@@ -0,0 +1,28 @@
+require 'csv'
```

Comment:
I've learned lots reading this, now I'll know how to load a CSV file in the future :)

Reply:
For the purpose of this exercise, a hash would've sufficed, but thumbs up for thinking of real life scenarios